src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -28,13 +28,7 @@ import type {
   CommercePaymentMethodEnum,
   Payment_order$data,
 } from "__generated__/Payment_order.graphql"
-import {
-  type FC,
-  type ReactElement,
-  type RefObject,
-  useEffect,
-  useRef,
-} from "react"
+import type { FC, ReactElement, RefObject } from "react"
```

Comment:
I'm so confused why it chose to add all this `type` and now insists on deleting? I assume it's prettier that is doing that?

Reply:
It seems it's moving `type` out of {} because `useEffect` and `useRef` are removed now.